app/tests/integration/nimbus/app_contexts.json

```diff
@@ -1,149 +1,144 @@
-      "data": {
-        "columns": [
-          {
-            "name": "app_context",
-            "friendly_name": "app_context",
-            "type": "string"
-          }
-        ],
-        "rows": [
-          {
-            "app_context": {
-              "app_name": "fenix",
-              "app_id": "org.mozilla.firefox",
-              "channel": "release",
-              "app_version": "95.2.0",
-              "app_build": "2015851759",
-              "architecture": "armeabi-v7a",
-              "device_manufacturer": "HUAWEI",
-              "device_model": "MED-LX9",
-              "locale": "es-MX",
-              "os": "Android",
-              "os_version": "10",
-              "android_sdk_version": "29",
-              "debug_tag": null,
-              "installation_date": 1641967200,
-              "home_directory": null,
-              "custom_targeting_attributes":
-                {
-                  "is_already_enrolled": "true",
-                  "days_since_update": "1",
-                  "days_since_install": "1"
-                }
+  "query_result": {
+    "id": 15734423,
```

Comment:
why is this changed?

Reply:
Oh vscode moved the indentation around

Reply:
The actual change was trying to get the `custom_targeting_attributes` to have the expected types, ie integers and bools, rather than strings.  But it turns out this whole thing gets parsed and expects only string types.  But you can pass in `custom_targeting_attributes` separately into the helper encoded as JSON with the actual types, so I ended up just taking it out of here entirely.